docs/guide/extending/extending_form_builder.md

```diff
@@ -199,13 +199,19 @@
     By adjusting them to your needs, you will change your email template.
 
     ### Configure sender details
-
-    To send emails you also need to configure `sender_address` in `app/config/config.yml`.
-    It acts as a sender address and a return address where all bounced messages will be returned to.
-    You can learn more by visiting [Symfony Mailer Configuration Reference.](https://symfony.com/doc/3.4/reference/configuration/swiftmailer.html#sender-address)
-
-    ### Configure email notifications
-
-    To send emails, you need to configure `sender_address` in `app/config/config.yml` under the `swiftmailer` key.
-    It acts as a sender and return address for all bounced messages.
-    For details, see [Symfony Mailer Configuration Reference.](https://symfony.com/doc/3.4/reference/configuration/swiftmailer.html#sender-address)
+    
+    To send emails from the Form Builder, you need to configure `sender_address` in `app/config/config.yml`. It acts as a sender and return address for all bounced messages. For more information, see [Symfony Mailer Configuration Reference.](https://symfony.com/doc/3.4/reference/configuration/swiftmailer.html#sender-address)
+        
+    Add `sender_address` entry to `app/config/config.yml`:
+    
+    ``` yaml
+    swiftmailer:
+        transport: '%mailer_transport%'
+        host: '%mailer_host%'
+        username: '%mailer_user%'
+        password: '%mailer_password%'
+        spool: { type: memory }
+        sender_address: 'mail@example.com'
```

Comment:
All other settings are using named parameters, it would be great if we can stick to that.
```suggestion
        sender_address: '%mailer_sender_address%`
```
And then mention, that this has to be set as well
`app/config/parameters.yml`:
```
parameters:
    mailer_sender_address: 'mail@example.com'
```

Reply:
Also, please add missing 
```
    host: '%mailer_host%'
```
to the example